_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Help! I'm becoming Post-Junior"
+date: 2016-10-10 12:17
+author: orta
+categories: [culture, juniors]
+series: Stages of Professional Growth
+---
+
+I’ve lived in NYC for 2 years now. This means I’ve been around long enough that some of the people I helped when they were learn to program have started to feel like they’re not “Juniors” anymore. 
+
+There is no arbitrary timeframe that you can say applies to that feeling of career progression, but from the explanations I’ve had recently - it’s been about feeling confident in your code, your impact and wanting to improve both.
```

Comment:
I think this paragraph could be expanded upon. It's only one sentence, but it's saying a lot.